src/routes/docs/advanced/security/+page.markdoc

```diff
@@ -31,6 +31,10 @@ Appwrite is SOC2 Type I compliant. Learn about Appwrite's measures to meet SOC 2
 {% cards_item href="/docs/advanced/security/hipaa" title="HIPAA" %}
 Appwrite is HIPAA compliant. Learn about Appwrite's measures to protect personal health information.
 {% /cards_item %}
+
+{% cards_item href="/docs/advanced/security/ccpa" title="CCPA" %}
```

Comment:
Let's check with design team if it's okay that there is a one cart missing, or if there is a better design we could do instead:

![CleanShot 2024-09-18 at 12 56 27@2x](https://github.com/user-attachments/assets/b680ee1f-bb45-4de2-aa31-8248b47d05db)

Reply:
Design approves the layout. The margin update, if needed, will come in another PR